qa/tasks/mgr/mgr_test_case.py

```diff
@@ -230,6 +231,18 @@ def _assign_ports(cls, module_name, config_name, min_port=7789):
         # Start handing out ports well above Ceph's range.
         assign_port = min_port
 
+        ip_addr = cls.mgr_cluster.get_mgr_map()['active_addr'].split(':')[0]
+
+        while True:
+            if not cls.is_port_in_use(ip_addr, assign_port):
+                break
+            log.debug("Port {0} in use, trying next".format(assign_port))
```

Comment:
If we don't use the new fancy f-strings, then I'd use the old `%` syntax:
```suggestion
            log.debug("Port %d in use, trying next", assign_port)
```

Reply:
went with the f-string.